src/GitHub/Data/PullRequests.hs

```diff
@@ -180,8 +184,22 @@ instance Binary PullRequestReference
 -- JSON instances
 -------------------------------------------------------------------------------
 
+-- | Helper function, reads either the "assignee" OR "assigneed" OR
+-- both from a JSON object.
+getAssignees :: Object -> Parser (Vector SimpleUser)
```

Comment:
@phadej if it's a dealbreaker for getting this merged then I can remove the `assignee` part